platform/security/filter/security-filter-web-sso/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.web</groupId>
+            <artifactId>pax-web-api</artifactId>
+            <version>6.0.6</version>
```

Comment:
❗️  We should define versions in the parent pom and reference them via variables here.

Reply:
I agree. Should be a property in the root DDF pom. I also would recommend using the latest version which looks like is `6.0.9` right now.